tests/integration/ceph_mgr_test.go

```diff
@@ -122,19 +119,61 @@ func (s *CephMgrSuite) execute(command []string) (error, string) {
 	return s.installer.Execute("ceph", orchCommand, s.namespace)
 }
 
+func (s *CephMgrSuite) enableRookOrchestrator() error {
+	logger.Info("Enabling Rook orchestrator module: <ceph mgr module enable rook --force>")
+	err, output := s.installer.Execute("ceph", []string{"mgr", "module", "enable", "rook", "--force"}, s.namespace)
+	logger.Infof("output: %s", output)
+	if err != nil {
+		logger.Infof("Not possible to enable rook orchestrator module: %q", err)
+		return err
+	}
+	logger.Info("Setting orchestrator backend to Rook .... <ceph orch set backend rook>")
+	err, output = s.execute([]string{"set", "backend", "rook"})
```

Comment:
Does this need to retry several times in case ceph hasn't initialized the module fully yet?

Reply:
The operation that really takes time is to have the rook toolbox ready to execute commands. Once available and enabled the rook module this operation (set backend rook) is really light and fast... .. only will fail if the rook manager module is not loaded , and i have only see that thing when the module has "syntax" errors.

Reply:
True, by the time the toolbox is ready everything else should be ready